.gitignore

```diff
@@ -9,4 +9,5 @@ dist
 obj
 docs/extensions
 .ipynb_checkpoints
-.idea
\ No newline at end of file
+.idea
+*_metadata.py
```

Comment:
I think this is not the right way to add this, because we want to keep the _metadata.py in the main directory

Reply:
ok! maybe better now?